src/vs/workbench/services/configuration/common/configuration.ts

```diff
@@ -5,7 +5,9 @@
 
 import { ConfigurationScope } from 'vs/platform/configuration/common/configurationRegistry';
 
+// {{SQL CARBON EDIT}}
 export const FOLDER_CONFIG_FOLDER_NAME = '.azuredatastudio';
+export const TASKS_FOLDER_CONFIG_FOLDER_NAME = '.vscode';
```

Comment:
why do we need .vscode folder?

Reply:
so users can seamlessly sync with their vscode tasks with ADS

Reply:
is this a requirement? it is confusing that azuredatastudio is actually relying on a .vscode folder.

Reply:
@abist any comment here?

Reply:
this was a follow up from the last time I presented the Task system for demo. I initially had it as .azuredatastudio